.github/workflows/deploy_artifacts_proxy.yaml

```diff
@@ -0,0 +1,62 @@
+name: Deploy Artifacts Proxy
```

Comment:
It scares me ever so slightly that we're hosting this in our public repo.  Any commit reviewed to artifacts_proxy could end up directly in prod...  Which I guess is good?  Just a bit scary to have it so little interaction to get to prod.

Reply:
Yeah we can make this a manual trigger only if you prefer.